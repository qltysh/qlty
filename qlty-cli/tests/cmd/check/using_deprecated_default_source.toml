bin.name = "qlty"
args = ["check", "--all", "--no-cache"]
