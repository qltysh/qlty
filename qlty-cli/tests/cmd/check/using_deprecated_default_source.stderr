
âš  Warning: qlty.toml is using a deprecated, repository-based, default source.

Please change the default source in your qlty.toml to:

[[source]]
name = "default"
default = true


   ERROR   

 > Please update your qlty.toml to use the new source format.
